api/v1/openapi.yaml

```diff
@@ -1,33 +1,33 @@
 ---
-swagger: '2.0'
+swagger: "2.0"
```

Comment:
Did your editor making these changes? we should avoid committing auto-linting changes, especially if they're not codified in the codebase.

Reply:
Rats! it did - will update

Reply:
This isn't resolved. There are still a lot of spurious changes; please revert all of these redundant changes.

Reply:
Rats! good catch. Tx Nate! tabs and valuable EOL spaces now restored to their former glory.